installer.py

```diff
@@ -230,40 +230,50 @@ def build(win, subitem, stdscr, colmid):
             if host_status == data_blob.cop_install_podnet_a:
-                cop_install_podnet_a.build(win)
+                built = cop_install_podnet_a.build(win)
             elif host_status == data_blob.cop_install_podnet_b:
-                cop_install_podnet_b.build(win)
+                built = cop_install_podnet_b.build(win)
             elif host_status == data_blob.cop_install_appliance_a:
-                cop_install_appliance_a.build(win)
+                built = cop_install_appliance_a.build(win)
             elif host_status == data_blob.cop_reinstall_podnet_a:
-                cop_reinstall_podnet_a.build(win)
+                built = cop_reinstall_podnet_a.build(win)
             elif host_status == data_blob.region_install_podnet_a:
-                region_install_podnet_a.build(win)
+                built = region_install_podnet_a.build(win)
             elif host_status == data_blob.region_install_podnet_b:
-                region_install_podnet_b.build(win)
+                built = region_install_podnet_b.build(win)
             elif host_status == data_blob.region_install_appliance_a:
-                region_install_appliance_a.build(win)
+                built = region_install_appliance_a.build(win)
             elif host_status == data_blob.region_reinstall_podnet_a:
-                region_reinstall_podnet_a.build(win)
+                built = region_reinstall_podnet_a.build(win)
             elif host_status == data_blob.copregion_install_podnet_a:
                 copregion_install_podnet_a.build(win)
             elif host_status == data_blob.copregion_install_podnet_b:
-                copregion_install_podnet_b.build(win)
+                built = copregion_install_podnet_b.build(win)
             elif host_status == data_blob.copregion_install_appliance_a:
-                copregion_install_appliance_a.build(win)
+                built = copregion_install_appliance_a.build(win)
             elif host_status == data_blob.copregion_reinstall_podnet_a:
-                copregion_reinstall_podnet_a.build(win)
+                built = copregion_reinstall_podnet_a.build(win)
             elif host_status == data_blob.pat_install_podnet_a:
-                pat_install_podnet_a.build(win)
+                built = pat_install_podnet_a.build(win)
             elif host_status == data_blob.pat_install_podnet_b:
-                pat_install_podnet_b.build(win)
+                built = pat_install_podnet_b.build(win)
             elif host_status == data_blob.pat_install_appliance_a:
-                pat_install_appliance_a.build(win)
+                built = pat_install_appliance_a.build(win)
             elif host_status == data_blob.pat_reinstall_podnet_a:
-                pat_reinstall_podnet_a.build(win)
+                built = pat_reinstall_podnet_a.build(win)
             else:
                 win.addstr(3, 1, f'Invalid Host status: {host_status}, cannot configure Pod', curses.color_pair(3))
```

Comment:
```suggestion
                built = False
                win.addstr(3, 1, f'Host {host_status}, is already in a configured state.', curses.color_pair(4))
```

I do not see this as an error as it will only enter here if the host status contains `validate` and is already configured.